docs/feature_mouse_keys.md

```diff
@@ -55,27 +62,55 @@ The default speed for controlling the mouse with the keyboard is intentionally s
 #define MOUSEKEY_WHEEL_TIME_TO_MAX 40
 ```
```

Comment:
I know this was already here, but since we're touching this documentation could you reformat this as a table with the following headers?

    Define, Default, Description

You can fold the descriptions below into it and make everything more compact and easier to read/find.